XML/Other/ReplaceableTestParameters.xml

```diff
@@ -227,6 +251,10 @@ Scenario 2 : You need to run all the performance tests quickly.
 		<ReplaceThis>INFINIBAND_OPEN_RMA_TEST_COUNT</ReplaceThis>
 		<ReplaceWith>0</ReplaceWith>
 	</Parameter>
+	<Parameter>
+		<ReplaceThis>INFINIBAND_HPCX_RMA_TEST_COUNT</ReplaceThis>
+		<ReplaceWith>0</ReplaceWith>
```

Comment:
Add comment here why we have to have 0 for now.

Reply:
added comment